dotnet/src/SemanticKernel.Abstractions/Functions/KernelFunctionArguments.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Collections.Generic;
+using Microsoft.SemanticKernel.AI;
+
+#pragma warning disable IDE0130
+namespace Microsoft.SemanticKernel;
+#pragma warning restore IDE0130
+
+/// <summary>
+/// Represents arguments of <see cref="KernelFunction.InvokeAsync"/> and "Kernel.InvokeAsync" methods.
+/// </summary>
+public sealed class KernelFunctionArguments : Dictionary<string, string>
+{
```

Comment:
I suggest there should also be a constructor that accepts a KernelFunctionArguments and another that accepts an `IDictionary<>`.

Reply:
I had both in my POC but wanted to add them along the way if they are needed. Anyway, they are added.